startup.sh

```diff
@@ -37,39 +38,42 @@ brew link node
 
 command_name=xcode-select
 if ! type "$command_name" > /dev/null; then
-    echo "\n${purple}Installing Xcode command line tools.${normal}\n"
+    echo -e "\n${purple}Installing Xcode command line tools.${normal}\n"
     $command_name --install
 else
-    echo "\n${purple}Xcode command line tools already exists, skipping install.${normal}\n"
+    echo -e "\n${purple}Xcode command line tools already exists, skipping install.${normal}\n"
 fi
 
-echo "\n${purple}Installing React Native dependencies using homebrew and npm.${normal}\n"
+echo -e "\n${purple}Installing React Native dependencies using homebrew and npm.${normal}\n"
 brew tap AdoptOpenJDK/openjdk
 brew cask install adoptopenjdk8
 npm install -g react-native-cli
 
-echo "\n\n${purple}Installing fbs/PyQt5 dependencies using pyenv and pip, this might take a while.${normal}\n\n"
+echo -e "\n\n${purple}Installing fbs/PyQt5 dependencies using pyenv and pip, this might take a while.${normal}\n\n"
 pyenv init
-echo "\n"eval '"$(pyenv init -)"' >> $rc_file
+echo -e "\n eval \"$(pyenv init -)\"" >> $rc_file
```

Comment:
Without the `-e` flag and the escaping, this was just adding
`\neval "$(pyenv init -)"` to the `~/.bashrc` file.
<img width="251" alt="Screen Shot 2019-08-11 at 5 34 04 PM" src="https://user-images.githubusercontent.com/507283/62839815-3e51b280-bc5e-11e9-82a4-65e18e16a29d.png">